modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflow.module

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Editorial Corporate Workflow module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\node\NodeInterface;
+
+/**
+ * Implements hook_menu_local_tasks_alter().
+ */
+function oe_editorial_corporate_workflow_menu_local_tasks_alter(&$data): void {
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node instanceof NodeInterface && !empty($data['tabs'][0]['entity.node.edit_form']['#link'])) {
+    /** @var \Drupal\content_moderation\ModerationInformationInterface $moderation_information */
+    $moderation_information = \Drupal::service('content_moderation.moderation_information');
+    if ($moderation_information->isModeratedEntity($node) &&
```

Comment:
Am I seeing this wrong or `$moderation_information->isModeratedEntity($node)` is duplicated in the conditional?